<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample database UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="outputframe">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QTextBrowser" name="outbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>256</width>
       <height>141</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="createplot">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addsample">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add sample to database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_file">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_pause_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test pause</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>User inputs</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QComboBox" name="n">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="film_density">
        <property name="minimum">
         <double>500.000000000000000</double>
        </property>
        <property name="maximum">
         <double>3000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>850.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="medium">
        <item>
         <property name="text">
          <string>air</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>water</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="f0label">
        <property name="text">
         <string>Resonant frequency</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;b&gt;Solution search space&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;b&gt;Sample properties&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="filmthicknesslabel">
        <property name="text">
         <string>Film thickness</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="f0unitslabel">
        <property name="text">
         <string>MHz</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="filmdensitylabel">
        <property name="text">
         <string>Film density</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Log(η): low - high</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="f0">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Pa s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="harmoniclabel">
        <property name="text">
         <string>Harmonic</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="bulkmediumlabel">
        <property name="text">
         <string>Bulk medium</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="filmdensityunits">
        <property name="text">
         <string>kg/m&lt;sup&gt;2&lt;/sup&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="film_thickness">
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>50000.000000000000000</double>
        </property>
        <property name="value">
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="filmthicknessunits">
        <property name="text">
         <string>nm</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="df_exp">
        <property name="minimum">
         <double>-500000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="df_label">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Δf</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="dd_units">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>x 10&lt;sup&gt;-6&lt;/sup&gt;</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="dd_label">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ΔD</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="df_units">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Hz/cm&lt;sup&gt;2&lt;/sup&gt;</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QDoubleSpinBox" name="dd_exp">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Log(μ): low - high</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Pa</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;b&gt;Measured values&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="mu_exp_low">
          <property name="minimum">
           <number>-2</number>
          </property>
          <property name="maximum">
           <number>17</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mu_exp_high">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>18</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="eta_exp_low">
          <property name="minimum">
           <number>-12</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="value">
           <number>-9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="eta_exp_high">
          <property name="minimum">
           <number>-11</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;b&gt;Crystal properties&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="run_model">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run model</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="show_contour_plots">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show contour plots</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShowfiledir"/>
    <addaction name="actionChangefiledir"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShowfiledir">
   <property name="text">
    <string>Show file save directory</string>
   </property>
  </action>
  <action name="actionChangefiledir">
   <property name="text">
    <string>Change file save directory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
